src/py21cmfast_tools/lc2ps.py

```diff
@@ -364,9 +484,77 @@ def postprocess_ps(
                 log_kpar[crop[2] : crop[3]],
                 nmodes,
             )
+        else:
+            return (
+                rebinned_ps[crop[0] : crop[1]][:, crop[2] : crop[3]],
+                kperp[crop[0] : crop[1]],
+                log_kpar[crop[2] : crop[3]],
+            )
     else:
         return (
             rebinned_ps[crop[0] : crop[1]][:, crop[2] : crop[3]],
             kperp[crop[0] : crop[1]],
             log_kpar[crop[2] : crop[3]],
         )
+
+
+def ps_2d21d(
```

Comment:
I **much** prefer more descriptive names without abusing "2" to mean "to", especially when "2" appears in the title anyway. So for this I would say a better name is `average_2d_ps_to_1d`, or `average_ps_cylindrical_to_spherical`

Reply:
Or even just `cylindrical_to_spherical`

Reply:
Not my choice but I concur. ```cylindrical_to_spherical``` seems brief and precise.